radix-engine/src/blueprints/account/package.rs

```diff
@@ -331,6 +334,7 @@ impl AccountNativePackage {
         }
     }
 
+    #[trace_resources]
```

Comment:
Why do we need macro for this method?

I thought the macro on `invoke_export` should cover all functions and methods.

Reply:
Yes, `trace_resources` macro covers whole `invoke_export` function, I just wanted to check if `create_global` takes the most instructions from `invoke_export`. This can be removed.